adagrams/game.py

```diff
@@ -1,13 +1,107 @@
 from random import randint
 
 def draw_letters():
-    pass
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    pile = []
+    hand = []
+    # created a list of letters
+    for k, v in LETTER_POOL.items():  
+        for counter in range(0, v):      
```

Comment:
Notice that `counter` is initialized but never accessed. When we write a loop but don't use the looping variable, we should use `_` instead. 

Also, we don't need to supply the beginning value for `range` when we will visit every value in a range so you can just have:
```suggestion
        for _ in range(v):      
```

Reply:
How could you use list comprehension to make this logic on lines 35-37 more concise?

```py
pile = [letter for letter, frequency in LETTER_POOL.items() for _ in range(frequency)]
```

Here's [documentation](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions) about comprehension for you to review, you'll see it a lot in Python!

Reply:
Prefer lines 35-37 be written in a helper function like `get_letter_pool` and then you'd just invoke that function here in `draw_letters`. 

Doing so would do a couple things: 
- make `draw_letters` more concise
- make `draw_letters` concerned with less logic
-  makes your code more self-documenting because instead of reading through lines of logic and surmising what it does, I would just be reading a function call that is very descriptive. 

So line 44 would like this after refactoring
```py
    pile = get_letter_pool()